### PR TITLE
Render accessibility statement from markdown

### PR DESCRIPTION
**Pull request by tijmenb**
### Context

We're going to have a number of content pages in the app, like the accessibility statement, privacy notice, etc. Having them rendered from markdown makes it easier for content designers to edit them.

### Changes proposed in this pull request

[@chubberlisk started this branch](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/compare/98-spike-redcarpet?expand=1).

This PR renders the accessibility statement from a markdown file. It uses a helper in the controller for that. I've renamed the controller to `ContentController` to make it suitable for future pages as well.

#### Before

| Before | After |
| --- | --- |
| image |  image |

### Guidance to review

Read the code.

### Link to Trello card

Slightly related to https://trello.com/c/gRCLMBqY.

### CODE REVIEW

File: app/views/content/rendered_markdown_template.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<%= content_for :title, page_title(params[:action]) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('page_titles.accessibility') %></h1>
```
Comment (tvararu) [inline app/views/content/rendered_markdown_template.html.erb]: Should this be in the template?

File: app/frontend/styles/application.scss [hunk 2]
```diff
@@ -16,3 +16,17 @@ $govuk-image-url-function: frontend-image-url;
 .autocomplete__wrapper, .autocomplete__input, .autocomplete__hint {
   font-family: $govuk-font-family;
 }
+
+.app-styled-content {
```
Comment (duncanjbrown) [inline app/frontend/styles/application.scss]: I don't understand this class name – what does it mean?

File: app/views/content/rendered_markdown_template.html.erb (hunk 1 above)
Comment (tijmenb) [reply app/views/content/rendered_markdown_template.html.erb]: Uh no.

File: app/controllers/content_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  def accessibility
+    render_content_page :accessibility
+  end
+
+private
+
+  def render_content_page(page_name)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML.new)
+
+    @converted_markdown = markdown.render(File.read("app/views/content/#{page_name}.md")).html_safe
```
Comment (duncanjbrown) [inline app/controllers/content_controller.rb]: Should we sanitize `page_name`?

Comment (tijmenb) [reply app/controllers/content_controller.rb]: It's set by us in the controller, so I don't think so?

File: app/frontend/styles/application.scss (hunk 2 above)
Comment (tvararu) [reply app/frontend/styles/application.scss]: It's to wrap HTML which contains plain tags such as `h1, h2, p` and automatically style them with GOV.UK typography, commonly used for convenience, in this case we're using it for the markdown section

Comment (tijmenb) [reply app/frontend/styles/application.scss]: `app` is the namespace for everything in this app (as opposed to `govuk`, which is CSS from govuk-frontend). I've moved this into its own file, maybe it's clearer?

File: app/controllers/content_controller.rb (hunk 3 above)
Comment (duncanjbrown) [reply app/controllers/content_controller.rb]: :+1: Pure paranoia at this stage

File: app/frontend/styles/application.scss (hunk 2 above)
Comment (duncanjbrown) [reply app/frontend/styles/application.scss]: Ah, `app-` is equivalent to `govuk-`. That helps, thank you :+1:

**Pull request merged**